calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorViewModelFactory.kt

```diff
@@ -0,0 +1,17 @@
+package camp.nextstep.edu.calculator
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import camp.nextstep.edu.calculator.domain.RecordRepository
+
+class CalculatorViewModelFactory(
+    private val recordRepository: RecordRepository,
+) : ViewModelProvider.Factory {
+    override fun <T : ViewModel> create(modelClass: Class<T>): T {
```

Comment:
```suggestion
class CalculatorViewModelFactory(
    context: Context,
) : ViewModelProvider.Factory {
    private val recordRepository: RecordRepository = Injector.provideRecordRepository(context)
    override fun <T : ViewModel> create(modelClass: Class<T>): T {
```

이런식으로 변경해보는 것은 어떨까요?
viewModel에 주입을 위한 코드기때문에 activity에서 미리 선언할 필요가 없습니다!

또한 Factory패턴의 특성상 한 가지 뷰모델 뿐만 아니라 여러가지의 뷰모델 중 한 가지를 반환해주는 상황이 생길 수 있는데요~
모든 viewModel이 같은 repository가 필요하지 않을 수 있습니다. 외부에서 주입받는 것은 조금 어색하다고 느껴지네요~